requirements.txt

```diff
@@ -1,9 +1,9 @@
-django>=1.11.16,<2.0
+django>=2.2,<3.0
```

Comment:
we could go up to 3 if we wanted now, right?

Reply:
Maybe, but it just came out in December and our current LTSA says we'll stick to LTS releases. I'm thinking we should stick to 2.2 for now.